src/main/webapp/js/chart-loader.js

```diff
@@ -22,19 +22,16 @@
             //TODO add the formatted values to msgRow array by using JS' push method
             msgData.addRow(msgRow);
         }
-        drawChart(msgData);
+        var chart = new google.visualization.BarChart(
```

Comment:
https://github.com/codeu-spring-2019-team-9/codeu-spring-2019-team-9/pull/17#commitcomment-32878337

Hopefully fixed the race condition